hooks/001-extra-packages.chroot

```diff
@@ -23,13 +23,14 @@ mkdir -p /var/log/journal
 apt update
 apt install --no-install-recommends -y ca-certificates
 
-# shellcheck disable=SC1091
-CODENAME=$(. /etc/os-release; echo "$UBUNTU_CODENAME")
-# enable the ucdev PPA with additional packages to build bases
-echo "deb https://ppa.launchpadcontent.net/ucdev/base-ppa/ubuntu/ $CODENAME main" \
-     > /etc/apt/sources.list.d/ubuntu-image.list
-
-cat >/etc/apt/trusted.gpg.d/ucdev-base-ppa.asc <<'EOF'
+if false; then
```

Comment:
Why are not we doing this anymore?

Reply:
The PPA does not exist yet. It would fail.

Reply:
Would be nice to open a PR readding this so it's visible

Reply:
Reading?

Reply:
Ah, re-adding. Well it will be needed once we need to backport some packages. Which is probably not going to happen until we get close to or passed 24.04. It does not need to be re-added if we do not have any package.

Reply:
Ah yes, excuse my lack of hyphen, that definitely reads wrong. I just wanted to make sure that it's added again if neccessary, but then a TODO is probably good enough,